.localizationrc.json

```diff
@@ -0,0 +1,14 @@
+{
+  "paths": {
+    "kbn": "src/core_plugins/kibana",
+    "commonServer": "src/server",
+    "commonUi": "src/ui",
+    "x_pack": "xpack/plugins"
```

Comment:
__issue:__ I guess it should point to a specific plugin? Like `"xpack.idx.mgmt": "xpack/plugins/index_management"` (or  `"xpack.idxMgmt": "xpack/plugins/index_management"`)?